spec/dummy/app/models/concerns/tenancy_concern.rb

```diff
@@ -0,0 +1,14 @@
+module TenancyConcern
```

Comment:
Is this not already a mixin in this project?

Reply:
not that I can see, the TenancyConcern here is specific to the sources api.